src/testHelpers/src/main/kotlin/org/vaccineimpact/orderlyweb/test_helpers/insertHelpers.kt

```diff
@@ -151,6 +151,33 @@ fun insertReport(name: String,
     }
 }
 
+fun insertReportWithPublishedAndUnpublishedVersions(name: String, 
+    publishedVersionId: String = "20211001-013233-72d597e9", 
+    unpublishedVersionId: String = "20211002-013233-72d597e9"
+){
+    insertReport(name, publishedVersionId)
+
+    JooqContext().use {
```

Comment:
Would be good to add a comment explaining why we can't just call `insertReport(name, unpublishedVersionId, false)` here - it's not obvious!

Reply:
I assumed you couldn't but it turns out you can. I've since decided to remove the helper entirely.